From 5fecaeabc0d8cf603230c56971c1617e6bb018d8 Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Wed, 27 Aug 2025 22:50:32 -0500
Subject: [PATCH] Change CosmicAppletWorkspaces-symbolic to
 CosmicAppletWorkspaces

This change matches upstream change 4dfb2025b60baa922933b62300f9da30c2a93eb8

Signed-off-by: Ryan Brue <ryanbrue.dev@gmail.com>
---
 cosmic-applets.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cosmic-applets.spec b/cosmic-applets.spec
index 4dd074c..95ab326 100644
--- a/cosmic-applets.spec
+++ b/cosmic-applets.spec
@@ -291,7 +291,7 @@ export VERGEN_GIT_SHA="%{commit}"
 %{_datadir}/icons/hicolor/scalable/apps/com.system76.CosmicAppletTiling.On.svg
 %{_datadir}/icons/hicolor/scalable/apps/com.system76.CosmicAppletTiling-symbolic.svg
 %{_datadir}/icons/hicolor/scalable/apps/com.system76.CosmicAppletTime-symbolic.svg
-%{_datadir}/icons/hicolor/scalable/apps/com.system76.CosmicAppletWorkspaces-symbolic.svg
+%{_datadir}/icons/hicolor/scalable/apps/com.system76.CosmicAppletWorkspaces.svg
 %{_datadir}/icons/hicolor/scalable/apps/com.system76.CosmicPanelAppButton.svg
 %{_datadir}/icons/hicolor/scalable/apps/com.system76.CosmicPanelWorkspacesButton.svg
 %{_datadir}/icons/hicolor/scalable/status/cosmic-applet-battery-display-brightness-high-symbolic.svg
-- 
2.51.0

